telegram/ext/commandhandler.py

```diff
@@ -159,3 +169,147 @@ def collect_optional_args(self, dispatcher, update=None, check_result=None):
 
     def collect_additional_context(self, context, update, dispatcher, check_result):
         context.args = check_result
+
+
+class PrefixHandler(CommandHandler):
```

Comment:
Should we add a docstring note about arguments being availible on the CallbackContext object? (applies to CommandHandler too I suppose)